.github/workflows/check.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
     with:
       additional-env-vars: |
         _R_CHECK_CRAN_INCOMING_REMOTE_=false
+        NOT_CRAN=true
```

Comment:
This is req. because without it, the `shinytests2` test will be skipped (even if there will be no `skip_on_cran`)